objects/src/accounts/code.rs

```diff
@@ -13,6 +13,16 @@ const ACCOUNT_CODE_TREE_DEPTH: u8 = 8;
 /// The maximum number of account interface procedures.
 const MAX_ACCOUNT_PROCEDURES: usize = 2_usize.pow(ACCOUNT_CODE_TREE_DEPTH as u32);
 
+// Default account code
+pub const DEFAULT_ACCOUNT_CODE: &str = "
+    use.miden::wallets::basic->basic_wallet
+    use.miden::eoa::basic->basic_eoa
+
+    export.basic_wallet::receive_asset
+    export.basic_wallet::send_asset
+    export.basic_eoa::auth_tx_rpo_falcon512
+";
+
```

Comment:
I think this should also go into `mock` crate as this is relevant only for mock/testing purposes.

Reply:
this is currently used in `miden-lib` [here](https://github.com/0xPolygonMiden/miden-base/pull/253/files#diff-b42801cd8387dd7cc03f6003e1966c4f0008a8b1fd3057ba0cd997e738c49fcfR20). To match the requested dependency graph this needs to stay here.

Reply:
This is for testing, right? If so, we should gate it by the "testing" feature for now and also probably leave a "TODO" to remove it from here once we move relevant tests from `miden-lib` to `miden-tx`.